tests/queries/0_stateless/02891_empty_tuple.sql

```diff
@@ -0,0 +1,13 @@
+drop table if exists x;
+
+create table x engine MergeTree order by () as select () as a, () as b;
+
```

Comment:
Does explicit type definition works?
```suggestion
create table y (a Tuple()) engine MergeTree order by a
```

Reply:
Added some more tests.